components/registry-facade-api/imagespec.proto

```diff
@@ -8,16 +8,18 @@ option go_package = "github.com/gitpod-io/gitpod/registry-facade/api";
 message ImageSpec {
     // base_ref points to an image in another registry
     string base_ref = 1;
-    // ide_ref points to an image denoting the IDE to use
+    // DEPRECATED ide_ref points to an image denoting the IDE to use
     string ide_ref = 2;
```

Comment:
With the deprecation of `ide_ref`, how does blobserve decide from which image it should serve files from?